frontend/test/metabase/scenarios/models/models-query-editor.cy.spec.js

```diff
@@ -135,6 +136,7 @@ describe("scenarios > models query editor", () => {
 
     cy.url().should("match", /\/model\/[1-9]\d*.*\d/);
     cy.url().should("not.include", "/query");
+    cy.location("hash").should("eq", "");
```

Comment:
@nemanjaglumac I also noticed a hash is appended to a URL after saving model query/metadata edits (see "To Verify" section or a demo video from the PR description). I've just added the hash assertions to query editor tests, not sure if I should extend the repro or leave it as is